code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -129,6 +129,7 @@
 		builtInCamera = new (src)
 		builtInCamera.c_tag = real_name
 		builtInCamera.network = list("SS13")
+		builtInCamera.internal_light = FALSE
```

Comment:
It looks like this should be TRUE

Reply:
This is correct. This PR is intended to prevent the AI turning on camera lights inside of a borg.